content/docs/user-guide/gems/reference/ros2/ros2-gem-user-guide.md

```diff
@@ -0,0 +1,221 @@
+---
+linkTitle: Guide
+title: ROS2 Gem Details
+description: User guide to develop ROS2 enabled simulation
+toc: true
+---
+
+The Gem and its build instructions can be found in [this repository](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2). The
+ROS 2 Gem helps to build robotic simulations with [ROS 2 / Robot Operating System](https://www.ros.org/). For an example
+of use see [Warehouse Demo Project](https://github.com/RobotecAI/Ros2WarehouseDemo) or [Robot Vacuum Sample](https://github.com/o3de/RobotVacuumSample).
+
+# Components overview
+
+- __Central Singleton__
+  - ROS2SystemComponent
+- __Core abstractions__
+  - ROS2FrameComponent
+  - ROS2SensorComponent
+- __Sensors__
+  - ROS2CameraSensorComponent
+  - ROS2GNSSSensorComponent
+  - ROS2IMUSensorComponent
+  - ROS2LidarSensorComponent
+  - ROS2OdometrySensorComponent
+- __Robot control__
+  - ROS2RobotControlComponent
+  - AckermannControlComponent
```

Comment:
Add Skid Steering component, in general update this once skid steering is merged (including Vehicle dynamics)